src/main/java/KontrolerDanych.java

```diff
@@ -0,0 +1,7 @@
+public class KontrolerDanych {
+    public KontrolerDanych() {
+
+    }
+
+
+}
```

Comment:
Ta klasa ma być pusta?

Reply:
póki co tak, bo nadal nie ogarniam jak to miało być xdd